packages/pages/src/upgrade/migrate_config.ts

```diff
@@ -65,10 +123,14 @@ export const migrateConfigs = async (target: string) => {
   }
   const configYamlPath = path.resolve(target, "config.yaml");
   if (!fs.existsSync(configYamlPath)) {
-    console.info("config.yaml does not exist, cannot migrate");
-    return;
+    console.info("config.yaml does not exist, creating it");
+    fs.writeFileSync(configYamlPath, "");
   }
-  await migrateGlobalData(configYamlPath, sitesConfigPath);
-  await migrateSiteMap(configYamlPath, sitesConfigPath);
+  await migrateCiJson(configYamlPath, sitesConfigPath);
```

Comment:
sitesConfigPath defined from line 119 needs to come from the projectStructure. Same for any other files.